ark/utils/plot_utils.py

```diff
@@ -6,13 +6,55 @@
 import skimage.io as io
 import matplotlib as mpl
 import matplotlib.pyplot as plt
+from mpl_toolkits.axes_grid1 import make_axes_locatable
 
 from skimage.segmentation import find_boundaries
 from skimage.exposure import rescale_intensity
 
 
 # plotting functions
 
+def plot_clustering_result(labeled_image, labels_dict, title,
+                           default_val=None, cmap='tab20', figsize=(10, 10)):
+    """Takes a labeled image and a dictionary that maps cell (labels) to clusters
+    and display an image showing which cells were assigned to which cluster.
+
+    Args:
+        labeled_image (numpy.ndarray):
+            2D numpy array of labeled cell objects.
+        labels_dict (dict):
+            a mapping between labeled cells and their clusters.
+        title (str):
+            The title of the image that will be displayed.
+        default_val (int):
+            Cells that were not assigned to any cluster (according to labels_dict
+            will be translated to this value. This value should not appear in
+            labels_dict.
+        cmap (str):
+            Cmap to use for the image that will be displayed.
+        figsize (tuple):
+            Size of the image that will be displayed.
+    """
```

Comment:
I generally like to leave a newline in between the docstring and the first line of code in a function, but I'm not sure if we're strictly enforcing this. @ngreenwald any comments?

Reply:
changed :)